JqueryLibrary/.gitignore

```diff
@@ -0,0 +1,2 @@
+package-lock.json
```

Comment:
We don’t need to put that in there! We leave it in the project to use the libraries whose versions are specified in package-lock.json and avoid conflicts.

Please, read about  package-lock.json file